contracts/QuestFactory.sol

```diff
@@ -484,6 +515,10 @@ contract QuestFactory is Initializable, LegacyStorage, OwnableRoles, IQuestFacto
         defaultMintFeeRecipient = mintFeeRecipient_;
     }
 
+    function setReferralRewardTimestamp(uint256 timestamp_) external onlyOwner {
```

Comment:
Need to remember to call this as part of the deployment. Another option is to actually just set this in the initialize call.

Reply:
It's a part of the `initialize` call already, but I think we are going to need to call this function on chains where we upgrade